app/src/main/java/io/intelehealth/client/activities/privacyNoticeActivity/PrivacyNotice_Activity.java

```diff
@@ -61,8 +64,42 @@ protected void onCreate(Bundle savedInstanceState) {
                 obj = new JSONObject(String.valueOf(FileUtils.encodeJSON(this, AppConstants.CONFIG_FILE_NAME)));
                 }
 
-            String privacy_string = obj.getString("privacyNoticeText");
-            privacy_textview.setText(privacy_string);
+            SharedPreferences sharedPreferences = getSharedPreferences("CommonPrefs", Activity.MODE_PRIVATE);

```

Comment:
We should use common pref which is already done.
we have common pref class that we have to use it don't create shared pref independently.